optuna/storages/_rdb/storage.py

```diff
@@ -271,10 +281,12 @@ def get_study_direction(self, study_id: int) -> StudyDirection:
         session = self.scoped_session()
 
         study = models.StudyModel.find_or_raise_by_id(study_id, session)
-        # Terminate transaction explicitly to avoid connection timeout during transaction.
```

Comment:
I think this comment is for `self._commit(session)` and it still exists. Could you tell me why you remove it?

Reply:
It's my mistake. I will revert the comment.